test/text-field.html

```diff
@@ -326,6 +334,70 @@
             textField.label = 0;
             expect(textField.hasAttribute('has-label')).to.be.true;
           });
+
+          it('setting helper updates has-helper attribute', function() {
+            textField.helperText = 'foo';
+            expect(textField.hasAttribute('has-helper')).to.be.true;
+          });
+
+          it('setting helper to empty string does not update has-helper attribute', function() {
+            textField.helperText = '';
+            expect(textField.hasAttribute('has-helper')).to.be.false;
+          });
+
+          it('setting helper to null does not update has-helper attribute', function() {
+            textField.helperText = null;
+            expect(textField.hasAttribute('has-helper')).to.be.false;
+          });
+
+          it('setting number helper updates has-helper attribute', function() {
+            textField.helperText = 0;
+            expect(textField.hasAttribute('has-helper')).to.be.true;
+          });
+
+          it('text-field with slotted helper updates has-helper attribute', function(done) {
+            const textFieldSlottedHelper = fixture('default-with-slotted-helper');
+            setTimeout(() => {
```

Comment:
I guess we can avoid timeouts by using similar approach as in `text-field` with dispatching `slot-change`:

https://github.com/vaadin/vaadin-text-field/blob/2c9c8aac97e16646d247d6646be7979770903541/test/custom-input.html#L79

Reply:
Done, except it didn't work with this test: https://github.com/vaadin/vaadin-text-field/blob/7cae1f2e3e6a3d7646bcabc80486a0b713a0f8fa/test/text-field.html#L379-L388

Reply:
Hmm...for me the same approach works in that test also